### ISSUE TITLE
Game from blue ray disc not displaying anymore after HFW/CFW installation

### ISSUE THREAD

**Issue by fillo-rinaldi** [ZHassanQ/PS3-Guide#2]
I have followed the guide but after flashing HFW and even after flashing CFW I’m unable to see the game that is inserted in the blue ray reader. Any suggestions ?

**Comment by ZHassanQ** [comment]
I don't understand what you mean. Do you mean after flashing (in HFW) and installing CFW your PS3 is unable to read Blu-ray Discs and before it was able to?

And also you can't flash in CFW, as you were warned that you will make permanent changes in your PS3 when you flashed it.

**Comment by fillo-rinaldi** [comment]
Exactly, before I was able to read blue ray games but now I can’t see the game icon even if the disc is inserted

**Comment by ZHassanQ** [comment]
What CFW firmware you downloaded?

If BD it means for broken blu-ray readers. (Same applies to BD-BT)

You can try reinstalling the firmware. And before it, check the integrity of the file.
